src/main/java/me/zeroeightsix/kami/module/modules/combat/Aura.java

```diff
@@ -23,20 +23,23 @@
 
 /**
  * Created by 086 on 12/12/2017.
- * Updated by hub on 27 October 2019
+ * Updated by hub on 31 October 2019
  */
 @Module.Info(name = "Aura", category = Module.Category.COMBAT, description = "Hits entities around you")
 public class Aura extends Module {
 
-    private Setting<Boolean> players = register(Settings.b("Players", true));
-    private Setting<Boolean> animals = register(Settings.b("Animals", false));
-    private Setting<Boolean> mobs = register(Settings.b("Mobs", false));
-    private Setting<Double> range = register(Settings.d("Range", 5.5d));
-    private Setting<Boolean> wait = register(Settings.b("Wait", true));
-    private Setting<Boolean> walls = register(Settings.b("Walls", false));
-    private Setting<Boolean> switchTo32k = register(Settings.b("32k Switch", false));
+    private Setting<Boolean> attackPlayers = register(Settings.b("Players", true));
+    private Setting<Boolean> attackMobs = register(Settings.b("Mobs", false));
+    private Setting<Boolean> attackAnimals = register(Settings.b("Animals", false));
+    private Setting<Double> hitRange = register(Settings.d("Hit Range", 5.5d));
+    private Setting<Boolean> ignoreWalls = register(Settings.b("Ignore Walls", true));
+    private Setting<WaitMode> waitMode = register(Settings.e("Mode", WaitMode.KAMI));
+    private Setting<Double> waitTick = register(Settings.d("Tick Wait", 3.0));
```

Comment:
Make this settings's visibility depend on `waitMode` being equal to `WaitMode.TICK` please

Reply:
not really sure how to implement this, can i change the visibility of a setting after registering it?

Reply:
The settings builders allow this functionality:

https://github.com/zeroeightysix/KAMI/blob/312074ad5f988b4801f712b948a47a745e36e43b/src/main/java/me/zeroeightsix/kami/module/modules/render/Brightness.java#L18

Reply:
changed [here](https://github.com/blockparole/KAMI/commit/67ff9e0496dbd9568a47ffb637c05761dd3f85ed)